Assets/DONT TOUCH/Enums/ItemType.cs

```diff
@@ -1,59 +1,60 @@
 ﻿public enum ItemType
 {
-    None = -1,
-    KeycardJanitor = 0,
-    KeycardScientist = 1,
-    KeycardResearchCoordinator = 2,
-    KeycardZoneManager = 3,
-    KeycardGuard = 4,
-    KeycardMTFPrivate = 5,
-    KeycardContainmentEngineer = 6,
-    KeycardMTFOperative = 7,
-    KeycardMTFCaptain = 8,
-    KeycardFacilityManager = 9,
-    KeycardChaosInsurgency = 10,
-    KeycardO5 = 11,
-    Radio = 12,
-    GunCOM15 = 13,
-    Medkit = 14,
-    Flashlight = 15,
-    MicroHID = 16,
-    SCP500 = 17,
-    SCP207 = 18,
-    Ammo12gauge = 19,
-    GunE11SR = 20,
-    GunCrossvec = 21,
-    Ammo556x45 = 22,
-    GunFSP9 = 23,
-    GunLogicer = 24,
-    GrenadeHE = 25,
-    GrenadeFlash = 26,
-    Ammo44cal = 27,
-    Ammo762x39 = 28,
-    Ammo9x19 = 29,
-    GunCOM18 = 30,
-    SCP018 = 31,
-    SCP268 = 32,
-    Adrenaline = 33,
-    Painkillers = 34,
-    Coin = 35,
-    ArmorLight = 36,
-    ArmorCombat = 37,
-    ArmorHeavy = 38,
-    GunRevolver = 39,
-    GunAK = 40,
-    GunShotgun = 41,
-    SCP330 = 42,
-    SCP2176 = 43,
-    SCP244a = 44,
-    SCP244b = 45,
-    SCP1853 = 46,
-    ParticleDisruptor = 47,
-    GunCom45 = 48,
-    SCP1576 = 49,
-    Jailbird = 50,
-    AntiSCP207 = 51,
-    GunFRMG0 = 52,
-    GunA7 = 53,
-    Lantern = 54,
+  None = -1,
+  KeycardJanitor = 0,
+  KeycardScientist = 1,
+  KeycardResearchCoordinator = 2,
+  KeycardZoneManager = 3,
+  KeycardGuard = 4,
+  KeycardMTFPrivate = 5,
+  KeycardContainmentEngineer = 6,
+  KeycardMTFOperative = 7,
+  KeycardMTFCaptain = 8,
+  KeycardFacilityManager = 9,
+  KeycardChaosInsurgency = 10,
+  KeycardO5 = 11,
+  Radio = 12,
+  GunCOM15 = 13,
+  Medkit = 14,
+  Flashlight = 15,
+  MicroHID = 16,
+  SCP500 = 17,
+  SCP207 = 18,
+  Ammo12gauge = 19,
+  GunE11SR = 20,
+  GunCrossvec = 21,
+  Ammo556x45 = 22,
+  GunFSP9 = 23,
+  GunLogicer = 24,
+  GrenadeHE = 25,
+  GrenadeFlash = 26,
+  Ammo44cal = 27,
+  Ammo762x39 = 28,
+  Ammo9x19 = 29,
+  GunCOM18 = 30,
+  SCP018 = 31,
+  SCP268 = 32,
+  Adrenaline = 33,
+  Painkillers = 34,
+  Coin = 35,
+  ArmorLight = 36,
+  ArmorCombat = 37,
+  ArmorHeavy = 38,
+  GunRevolver = 39,
+  GunAK = 40,
+  GunShotgun = 41,
+  SCP330 = 42,
+  SCP2176 = 43,
+  SCP244a = 44,
+  SCP244b = 45,
+  SCP1853 = 46,
+  ParticleDisruptor = 47,
+  GunCom45 = 48,
+  SCP1576 = 49,
+  Jailbird = 50,
+  AntiSCP207 = 51,
+  GunFRMG0 = 52,
+  GunA7 = 53,
+  Lantern = 54,
+  SCP1344 = 55,
```

Comment:
Just added SCP1344, the file diff doesn't show it due different spacing which i didn't noticed